grpc/src/main/java/com/linecorp/armeria/internal/server/grpc/AbstractServerCall.java

```diff
@@ -137,7 +137,7 @@
     private volatile boolean listenerClosed;
     private boolean closeCalled;
 
-    AbstractServerCall(HttpRequest req,
+    public AbstractServerCall(HttpRequest req,
                        MethodDescriptor<I, O> method,
```

Comment:
nit: indentation
```suggestion
                              MethodDescriptor<I, O> method,
```

Reply:
Done in [fa4d628](https://github.com/line/armeria/commit/fa4d628b0a113437da1d9ebb625332dad3b299be)!